service_worker_api/index.d.ts

```diff
@@ -2,245 +2,13 @@
 // Project: https://developer.mozilla.org/fr/docs/Web/API/ServiceWorker_API
 // Definitions by: Tristan Caron <https://github.com/tristancaron>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
+// TypeScript Version: 2.1
 
 // <reference path="../es6-promise/es6-promise.d.ts" /> // REMOVED third "/" so this doesn't fire. Problem with duplicate Promises
+// <reference path="../whatwg-fetch/index.d.ts" />
```

Comment:
This is only a comment like the above line. I think we shouldn't import `whatwg-fetch` here, for it may cause some potential conflicts, eg. when user want to use `isomorphic-fetch` with this lib.